src/codegen/mod.rs

```diff
@@ -1560,6 +1561,33 @@ fn raw_type(ctx: &BindgenContext, name: &str) -> P<ast::Ty> {
     }
 }
 
+fn float_kind_rust_type(ctx: &BindgenContext, fk: FloatKind) -> P<ast::Ty> {
```

Comment:
Probably we could move this (and `raw_type`) to the `helper` module, if you wish. Otherwise it's fine :).